assets/src/dashboard/components/detailViewNavBar/index.js

```diff
@@ -89,16 +89,18 @@ export function DetailViewNavBar({ handleCta, handleBookmarkClick, ctaText }) {
         {handleBookmarkClick && (
           <BookmarkToggle onClick={handleBookmarkClick} />
         )}
-        <CapitalizedButton type={BUTTON_TYPES.CTA} onClick={handleCta}>
-          {ctaText}
-        </CapitalizedButton>
+        {ctaText && ctaText.trim().length > 0 && (
+          <CapitalizedButton type={BUTTON_TYPES.CTA} onClick={handleCta}>
+            {ctaText}
+          </CapitalizedButton>
+        )}
```

Comment:
Sometimes we might not want a call to action, like when data is missing, but we still want the base title bar with close button. This is a clean way to prevent the cta from showing.